buildpack.md

```diff
@@ -1102,17 +1098,23 @@ The `[[buildpack.licenses]]` table is optional and MAY contain a list of buildpa
 *Key: `sbom-formats = [ "<string>" ]`*
  - MUST be supported SBOM media types as described in [Software-Bill-of-Materials](#software-bill-of-materials).
 
-#### Stacks
+#### Targets
```

Comment:
I don't know much about the buildpack registry - is there anything that needs to change in distribution for this? Would `pack buildpack publish` for a buildpack put artifacts where they belong given these new rules?

Reply:
@jkutner do you have insight into this?

Reply:
For distribution + registry, everything is based off of the buildpack id. I don't believe we ever stored the "stack" looking through the [buildpack registry spec](https://github.com/buildpacks/spec/blob/main/extensions/buildpack-registry.md) I also didn't find stack mentioned anywhere.